nixos/tests/pam/test_chfn.py

```diff
@@ -0,0 +1,24 @@
+expected_lines = {
+    "account required pam_unix.so",
+    "account sufficient /nix/store/wba7p4ks843gb8178vwi17vq11ngilfb-pam-krb5-4.10/lib/security/pam_krb5.so",
```

Comment:
this will change once the derivation changes

you'd need to dynamically get the path of the derivation and then put it in here

```
testScript = builtins.replaceStrings [ "@@pam@@" ] [ pkgs.pam ] (builtins.readFile ./test_chfn.py)
```

Reply:
How's this?

Reply:
Perfect